app/assets/stylesheets/application.scss

```diff
@@ -27,6 +27,8 @@
   }
 }
 
+$govuk-page-width: 1140px;
```

Comment:
I believe it needs to be at the top @davidgisbey.

Also the standard pattern is to put all variables at the top of the file as well.

Reply:
Okay great thanks. Have made the change